client/src/api/TestSessionApi.ts

```diff
@@ -70,15 +69,18 @@ export function getTestSessionData(test_session_id: string): Promise<TestOutput[
     });
 }
 
-function assignParentGroups(session: TestSession): TestSession {
+/* Populate additional properties for API results */
+function addProperties(session: TestSession): TestSession {
   let groups = session.test_suite.test_groups;
   if (groups) {
-    groups = assignParentGroupsHelper(groups, null);
+    groups = assignParentGroups(groups, null);
+    groups = expandRunAsGroupChildren(groups, false);
   }
   return session;
 }
 
-function assignParentGroupsHelper(groups: TestGroup[], parent: TestGroup | null): TestGroup[] {
+// Can be safely removed
```

Comment:
What can be safely removed?

Reply:
Logic to add parent groups to API results